src/main/java/seedu/address/model/recipe/attribute/IngredientList.java

```diff
@@ -60,22 +41,11 @@ public void addIngredient(Ingredient ingredient) {
      * @param ingredient the ingredient to be deleted.
      */
     public void deleteIngredient(Ingredient ingredient) {
```

Comment:
Feel like we should implement a 'delete ingredient @index' method as well.

Reply:
Yup, that sounds a useful method